worlds/witness/docs/en_The Witness.md

```diff
@@ -25,7 +25,7 @@ This includes symbols such as "Dots", "Black/White Squares", "Colored Squares",
 
 Alternatively (or additionally), you can play "Door shuffle", where some doors won't open until you receive their "key".
 
-Receiving lasers as items is also a possible setting.
+Receiving lasers as items is also a possible option.
```

Comment:
Oooo I don't like this, this massively overlaps with the colloquial definition of "option"
Maybe this can be reworded to something like "is also an option you can choose"?

(Tbh this is part of the reason I would've preferred "settings" as the new catch-all term, but eh)

Reply:
```suggestion
You can also chose to receive lasers as items as an option.
``` how does this sound?

Reply:
Sure, either of those work for me, I probably thing Rever's is better? Otherwise someone might think they can do some in-game action for it or sth

Reply:
Scipio caught a typo tho o: